src/components/ContainerHomePreview.react.js

```diff
@@ -29,15 +29,6 @@ var ContainerHomePreview = React.createClass({
     clearInterval(this.timer);
   },
 
-  handleClickPreview: function () {
-    if (this.props.defaultPort) {
-      metrics.track('Opened In Browser', {
-        from: 'preview'
-      });
-      shell.openExternal('http://' + this.props.ports[this.props.defaultPort].url);
-    }
-  },
-
   handleClickNotShowingCorrectly: function () {
```

Comment:
@FrenchBen I'm so glad you suggested that. I was reluctant to rename it myself as much as I wanted to.

Fixed and squashed.